lit_gpt/data/tinystories.py

```diff
@@ -0,0 +1,181 @@
+"""https://github.com/karpathy/llama2.c/blob/b3c4b6/tinystories.py"""
+
+import glob
+import json
+import os
+import random
+from concurrent.futures import ProcessPoolExecutor
+from dataclasses import dataclass, field
+from functools import partial
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+import torch
+from torch.utils.data import ConcatDataset, DataLoader
+from tqdm import tqdm
+
+from lit_gpt.data.alpaca import download_if_missing
+from lit_gpt.data.base import LitDataModule
+from lit_gpt.tokenizer import Tokenizer
+
+
+@dataclass
+class TinyStories(LitDataModule):
```

Comment:
Could you follow the OpenWebText data module for the implementation? This would streamline the data loading and so we don't have to reimplement the chunking / packed-dataset which we had before.

Reply:
Do you mean using litdata? I thought we wanted to keep these bits of the karpathy code